app/auth/AuthorisedActions.scala

```diff
@@ -17,12 +17,12 @@
 package auth
 
 import play.api.Logging
-import play.api.libs.json.Reads
-import play.api.mvc.{ActionBuilder, _}
+import play.api.mvc._
 import repositories.MissingCTDocument
 import uk.gov.hmrc.auth.core.AuthProvider.GovernmentGateway
-import uk.gov.hmrc.auth.core.retrieve.{Retrieval, SimpleRetrieval, ~}
-import uk.gov.hmrc.auth.core.{AuthorisationException, _}
+import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
+import uk.gov.hmrc.auth.core.retrieve.{Retrieval, ~}
```

Comment:
I think this is still using v1 retrievals for the most part unless I'm missing something?

Reply:
It's only the `Retrievals` object _(plus a couple of other things that are used by this service)_ which was moved to a `v2` package:
![Screenshot 2022-09-01 at 12 02 32](https://user-images.githubusercontent.com/16954819/187899159-50a39f51-452b-4394-a3c7-4749eaf3b9f8.png)

So, this is the only change that's required. To swap out the Retrievals object for the `v2.Retrievals` object.